bears/general/LineCountBear.py

```diff
@@ -10,14 +10,24 @@ class LineCountBear(LocalBear):
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Formatting'}
 
-    def run(self, filename, file, max_lines_per_file: int):
+    def run(self, filename, file, max_lines_per_file: int,
+            exclude_blank_lines: bool=False):
         """
         Count the number of lines in a file and ensure that they are
         smaller than a given size.
 
         :param max_lines_per_file: Number of lines allowed per file.
+        :param exclude_blank_lines: Enable to exclude blank lines.
         """
         file_length = len(file)
+
+        blank_line_counter = 0
+        file_length = len(file)
+        for line in file:
+            if line.split() == []:
+                blank_line_counter += 1
```

Comment:
for this 
import re
a = ['line 1', 'line 2', 'line 3','\n','\t line 4',' line 5','     ', '\n\n']

lines=list(filter(lambda x: re.match(r'\s+',x),a))
print(lines)

the output is
['\n', '\t line 4', ' line 5', '     ', '\n\n']

and for x.strip()
a = ['line 1', 'line 2', 'line 3','\n','\t line 4',' line 5','     ', '\n\n']

lines=list(filter(lambda x: x.strip(),a))
print(lines)
the output is
['line 1', 'line 2', 'line 3', '\t line 4', ' line 5']